src/fidesops/models/fidesops_user.py

```diff
@@ -0,0 +1,50 @@
+from typing import Dict, Any
+
+from sqlalchemy import Column, String
+from sqlalchemy.orm import Session, relationship
+
+from fidesops.core.config import config
+from fidesops.db.base_class import Base
+from fidesops.util.cryptographic_util import generate_salt, hash_with_salt
+
+
+class FidesopsUser(Base):
+    """The DB ORM model for FidesopsUser"""
+
+    username = Column(String, unique=True, index=True)
+    hashed_password = Column(String, nullable=False)
+    salt = Column(String, nullable=False)
+
+    client = relationship(
+        "ClientDetail", backref="user", cascade="all, delete", uselist=False
+    )
+
+    @classmethod
+    def create(cls, db: Session, data: Dict[str, Any]) -> "FidesopsUser":
+        """Create a FidesopsUser by hashing the password with a generated salt
+        and storing the hashed password and the salt"""
+        salt = generate_salt()
+        hashed_password = hash_with_salt(
+            data["password"].encode(config.security.ENCODING),
+            salt.encode(config.security.ENCODING),
+        )
+
+        user = super().create(
+            db,
+            data={
+                "salt": salt,
+                "hashed_password": hashed_password,
+                "username": data["username"],
```

Comment:
Should the username be encrypted? 

I am thinking someone might use their real name as a username. 
On the Pro Platform we use application level encryption on PII that goes into the DB. But I am not sure if we are carrying over the same security policies from Pro. Or if a username should be treated as PII.

Reply:
I'm not sure either, I didn't treat it as PII since the requirements specified just `password` https://github.com/ethyca/fidesops/issues/240, but this may be something that was overlooked.  @seanpreston, @adriaaaa can you weigh in if admin UI usernames should be stored encrypted?

Reply:
discussed in person, we'll not encrypt usernames for now, but a change made to not log them, logging user id instead